src/sql/parts/modelComponents/tree/tree.component.ts

```diff
@@ -123,7 +123,7 @@ export default class TreeComponent extends ComponentBase implements IComponent,
 	/// IComponent implementation
 
 	public layout(): void {
-		this._changeRef.detectChanges();
+		super.layout();
```

Comment:
Again should this be put after the tree layout? Or does it not matter since this just tags it for change detection on next tick?